File: LATIHAN2DPBO2023/Python/Main.py
# Saya Mochamad Khaairi NIM 2106416 mengerjakan Latihan 2
# dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
# untuk keberkahanNya maka saya tidak melakukan kecurangan 
# seperti yang telah dispesifikasikan. Aamiin.

# import library
from Pilih import Pilih

# deklarasi list
mmahasiswa = []
n = 0
flag = True

# menu pilihan (selama pilihan bukan 3)
while flag:
	# inputan untuk pilihan
	print("\nMasukkan menu (1 : Input, 2 : Tampil, 3 : Keluar):")
	n = int(input())

 	# jika pilihan 1 maka tambah data
	if(n == 1):
		Pilih.add(mmahasiswa)

	# jika pilihan 2 maka tampilkan data
	if(n == 2):
		Pilih.show(mmahasiswa)

	# jika pilihan 3 maka keluar dari program
	if(n == 3):
		flag = False

File: LATIHAN2DPBO2023/CPP/Main.cpp
/*Saya Mochamad Khaairi NIM 2106416 mengerjakan Latihan 2
dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan 
seperti yang telah dispesifikasikan. Aamiin.*/

//import library
#include <bits/stdc++.h>
#include "Pilih.cpp"

using namespace std;

int main()
{
	//instance object menggunakan class Pilih
	Pilih pilihan;
	//deklarasi list
	list<Mahasiswa> llist;
	int n = 0;
	//menu pilihan
	//perulangan selama inputan pemilihan menu bukan 3 
	do
	{
		//masukkan untuk pilihan menu
		cout << "\nMasukkan menu (1 : Input, 2 : Tampil, 3 : Keluar):" << '\n';
		cin >> n;
		//pengecekan pilihan
		if(n != 3)
		{
			//pilihan 1 untuk menambahkan data
			if(n == 1)
			{
				pilihan.add(&llist);
			}

			//pilihan 2 untuk menampilkan data
			if(n == 2)
			{
				pilihan.show(&llist);
			}
		}
	}while(n != 3);
}